firebase-abt/src/main/java/com/google/firebase/abt/FirebaseABTesting.java

```diff
@@ -140,6 +140,52 @@ public void removeAllExperiments() throws AbtException {
     removeExperiments(getAllExperimentsInAnalytics());
   }
 
+  /**
+   * Adds an experiment to be active in GA by setting a null triggering condition on the provided
+   * experiment. This results in the experiment being active as if it was triggered by the
+   * triggering condition event being seen in GA.
+   *
+   * <p>Note: This is a blocking call and therefore should be called from a worker thread.
+   *
+   * @param activeExperiment The {@link AbtExperimentInfo} that should be set as active in GA.
+   * @throws AbtException If there is no Analytics SDK.
+   */
+  @WorkerThread
+  public void reportActiveExperiment(AbtExperimentInfo activeExperiment) throws AbtException {
+    throwAbtExceptionIfAnalyticsIsNull();
+    ArrayList<AbtExperimentInfo> activeExperimentList = new ArrayList<>();
+
+    // Remove trigger event if it exists, this sets the experiment to active.
+    Map<String, String> activeExperimentMap = activeExperiment.toStringMap();
+    activeExperimentMap.remove(AbtExperimentInfo.TRIGGER_EVENT_KEY);
+
+    // Add experiment to GA
+    activeExperimentList.add(AbtExperimentInfo.fromMap(activeExperimentMap));
+    addExperiments(activeExperimentList);
+  }
+
+  /**
+   * Cleans up all experiments which are active in GA but not currently running.
+   *
+   * <p>Note: This is a blocking call and therefore should be called from a worker thread.
```

Comment:
note here maybe that it overrides the full running list?
this comment would be confusing w.r.t 'reportActive'
here we should only 'clean up experiments which are noted in GA but not eligible for this device'? (running might be confused with 'active')

Reply:
updated